### DISCUSSION TITLE
`--wheres` syntax with `pscale database dump`

### DISCUSSION THREAD

**Discussion by kellysutton** [planetscale/discussion#637] [FAQs] [answered]
Hello!

Noticed what seems to be a relatively new `--wheres` flag with the `dump` command. Not finding any documentation for what the syntax of that might look like.

Could you point me toward the docs or alternatively provide a few examples?

Thanks!

**Comment by mscoutermarsh** [comment, marked as answer]
@kellysutton! Hello friend.

Both `--tables` and `--wheres` are comma delimited strings. You can use them like this...

```
pscale database dump my-db main --tables "users" --wheres "organization_id = 1"
```

That will only dump records from the `users` table `where organization_id = 1`.

Then if you want to do multiple...

```
pscale database dump my-db main --tables "users, items" --wheres "organization_id = 1, organization_id = 1"
```

**Comment by kellysutton** [comment]
Great, thank you for the response!
